lib/components/atoms/Button/tests/__snapshots__/Button.style.test.js.snap

```diff
@@ -172,29 +181,29 @@ exports[`Button should render correctly 1`] = `
         "SHADOW_S": "var(--shadow-s)",
       },
       "spacing": Object {
-        "HALF_UNIT": "0.25rem",
-        "MAX_CONTENT": "80rem",
-        "QUAD_UNIT": "0.125rem",
-        "UNIT_1": "0.5rem",
-        "UNIT_10": "5rem",
-        "UNIT_11": "5.5rem",
-        "UNIT_12": "6rem",
-        "UNIT_13": "6.5rem",
-        "UNIT_14": "7rem",
-        "UNIT_15": "7.5rem",
-        "UNIT_16": "8rem",
-        "UNIT_17": "8.5rem",
-        "UNIT_18": "9rem",
-        "UNIT_19": "9.5rem",
-        "UNIT_2": "1rem",
-        "UNIT_20": "10rem",
-        "UNIT_3": "1.5rem",
-        "UNIT_4": "2rem",
-        "UNIT_5": "2.5rem",
-        "UNIT_6": "3rem",
-        "UNIT_7": "3.5rem",
-        "UNIT_8": "4rem",
-        "UNIT_9": "4.5rem",
+        "halfUnit": "0.25rem",
```

Comment:
Your PR is actually reverting my good fixes to old and problematic state. 

Why have you started developing on first week's branch. Please create a fresh branch from develop and move only your changes there